world/dominion/general_dominion_commands.py

```diff
@@ -2537,8 +2536,12 @@ def func(self):
                 if num >= max_p:
                     caller.msg("You already have the maximum number of proteges for your social rank.")
                     return
-                if not self.check_social_rank_difference(char):
+                rankcheck, diff = self.check_social_rank_difference(caller.char_ob,char)
+                if not rankcheck: 
+                    self.msg("Your social rank must be at least %d higher than your target." % diff)                                      
                     return
+                else:
+                    pass
```

Comment:
Was this a placeholder you forgot to remove?

Reply:
I can honestly not remember why I wrote it like that.